src/Raven.Studio/wwwroot/App/views/database/status/storageReport.html

```diff
@@ -7,53 +7,58 @@
         </div>
         <div class="chart"></div>
     </div>
-    <div class="current-path">
-        <span data-bind="foreach: currentPath">
-            <a href="#" data-bind="click: _.partial($root.onClick, $data, false), attr: { class: type }">
-                <small data-bind="text: type"></small>
-                <span data-bind="text: name"></span>
-            </a>
-            <i data-bind="visible: $index() < $root.currentPath().length - 1" class="icon-arrow-filled-right"></i>
-        </span>
-    </div>
-    <div class="flex-horizontal flex-noshrink" data-bind="visible: accessManager.canCompactDatabase">
+    <div class="flex-horizontal">
+        <div class="current-path">
+            <span data-bind="foreach: currentPath">
+                <a href="#" data-bind="click: _.partial($root.onClick, $data, false), attr: { class: type }">
+                    <small data-bind="text: type"></small>
+                    <span data-bind="text: name"></span>
+                </a>
+                <i data-bind="visible: $index() < $root.currentPath().length - 1" class="icon-arrow-filled-right"></i>
+            </span>
+        </div>
         <div class="flex-separator"></div>
-        <button class="btn btn-sm btn-default" data-bind="click: compactDatabase">
-            <i class="icon-compact"></i><span>Compact Database</span>
-        </button>
+        <div class="padding padding-xs margin-right margin-right-sm" title="Set collections to be compressed - Navigate to Document Compression view">
+            <small>
+                <a target="_blank" data-bind="attr: { href: documentsCompressionUrl }"><i class="icon-link margin-right margin-right-xs"></i>Document Compression</a>
```

Comment:
I think it shoun't be target blank? It isn't external link, right?

Reply:
done